packages/dw-react-controls/src/components/PeoplePicker/models/GroupType.ts

```diff
@@ -0,0 +1,18 @@
+
+/**
+ * Possible variations of group types in the Microsoft ecosystem.
+ */
+export enum GroupType {
```

Comment:
Should we make this a flagged enum? Might need to support combination of choices.

Reply:
You can now provide it as an array (e.g. [GroupType.AAD, GroupType.M365]). 
Maybe it is indeed a bit more elegant to use the flagged enum